feat(changelog): secret-manager-changed-a-new-version-of-secret-manager- 2024-02-28 (#2811)
scaleway/docs-content: changelog/february2024/2024-02-28-secret-manager-changed-a-new-version-of-secret-manager-.mdx

---

> title: A new version of Secret Manager API is available

Reviewer: ```suggestion
title: A new version of the Secret Manager API is available
```

status: changed
author:
    fullname: 'Join the #secret-manager channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-02-28
category: security-identity
product: secret-manager
---

> Scaleway Manager API now has a new version available: v1beta1.

Reviewer: ```suggestion
Scaleway's Secret Manager has a new API version available: `v1beta1` !
```

This new version comes with major improvements:
- Improved resilience and scalability
- No more folder endpoints, but still the possibility to use paths to organize secrets

> - Simplicity to change the path of a secret

Reviewer: ```suggestion
- Changing the path of a secret has become simpler
```

> Scaleway public console now uses this version for Secret Manager product pages. For more information, you can read the <Link to=https://www.scaleway.com/en/developers/api/secret-manager/>API documentation website</Link>

Reviewer 2: ```suggestion
Scaleway public console now uses this version for Secret Manager product pages. For more information, you can read the [API documentation website](https://www.scaleway.com/en/developers/api/secret-manager).
```

> Please note that the v1alpha1 API still is supported but will be eventually deprecated.

Reviewer 2: ```suggestion
Note that the v1alpha1 API still is supported but will be eventually deprecated.
```
